.gitignore

```diff
@@ -89,24 +102,15 @@ tags
 
 ### VisualStudioCode ###
 .vscode/*
-
-# End of https://www.gitignore.io/api/go,vim,emacs,visualstudiocode
-
-# Build artifacts
-/build
-test/ansible-operator/ansible-operator
-test/helm-operator/helm-operator
-images/scorecard-proxy/scorecard-proxy
-images/scorecard-test/scorecard-test
-images/scorecard-test-kuttl/scorecard-test-kuttl
-
-# Test artifacts
-internal/ansible/runner/testdata/valid.yaml
-/bin
-internal/scorecard/examples/custom-scorecard-tests/go.sum
-
-# Website
-website/public/
-website/resources/
-website/node_modules/
-website/tech-doc-hugo
```

Comment:
should not it still? When the hugo commands are executed locally are not these dirs generated?

Reply:
These are just moved to the top of the file to group all of our custom rules together, separate from the generated gitignore.io rules.